samples/hello-world/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "bigtable-hello-world-node",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {
+    "@google-cloud/bigtable": "^0.10.2",
```

Comment:
Does it make sense to update this to the latest version?

Reply:
Updated to the latest (`v0.13.1`)